<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerSetupDialog</class>
 <widget class="QDialog" name="PlayerSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QListView" name="playerListview"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="addPlayerBtn">
         <property name="text">
          <string>New Player</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removePlayerBtn">
         <property name="text">
          <string>Remove Player</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="playerSetupBox">
       <property name="title">
        <string>Connection Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Player Name</string>
            </property>
            <property name="buddy">
             <cstring>playerName</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="playerName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>IP Address</string>
            </property>
            <property name="buddy">
             <cstring>playerHost</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="playerHost"/>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Username</string>
            </property>
            <property name="buddy">
             <cstring>playerUser</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="playerUser">
            <property name="text">
             <string>player</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Password</string>
            </property>
            <property name="buddy">
             <cstring>playerPass</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLineEdit" name="playerPass">
            <property name="text">
             <string>player</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="playerOutputBox">
       <property name="title">
        <string>Output Options</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Screen Position:</string>
          </property>
          <property name="buddy">
           <cstring>outputX</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>X</string>
            </property>
            <property name="buddy">
             <cstring>outputX</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="outputX">
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="minimum">
             <number>-999</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Y</string>
            </property>
            <property name="buddy">
             <cstring>outputY</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="outputY">
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="minimum">
             <number>-999</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Width</string>
            </property>
            <property name="buddy">
             <cstring>outputWidth</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="outputWidth">
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="minimum">
             <number>-999</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>1024</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Height</string>
            </property>
            <property name="buddy">
             <cstring>outputHeight</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="outputHeight">
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="minimum">
             <number>-999</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>768</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="optFlipV">
          <property name="text">
           <string>Flip Image Vertically</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="optIgnoreAR">
          <property name="text">
           <string>Ignore Viewport Aspect Ratio</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="optFlipH">
          <property name="text">
           <string>Flip Image Horizontally</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>label_5</zorder>
       <zorder>horizontalLayoutWidget</zorder>
       <zorder>optFlipV</zorder>
       <zorder>optIgnoreAR</zorder>
       <zorder>platyerAdvancedBox</zorder>
       <zorder>optFlipH</zorder>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="platyerAdvancedBox">
       <property name="title">
        <string>Advanced Options</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Content Viewport</string>
          </property>
          <property name="buddy">
           <cstring>viewportX</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>X</string>
            </property>
            <property name="buddy">
             <cstring>viewportX</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="viewportX"/>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Y</string>
            </property>
            <property name="buddy">
             <cstring>viewportY</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="viewportY"/>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Width</string>
            </property>
            <property name="buddy">
             <cstring>viewportWidth</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="viewportWidth"/>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Height</string>
            </property>
            <property name="buddy">
             <cstring>viewportHeight</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="viewportHeight"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Advanced Viewports (Subviews)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QListWidget" name="listWidget"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="addSubviewBtn">
               <property name="text">
                <string>Add Subview</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="delSubviewBtn">
               <property name="text">
                <string>Remove Subview</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Subview Settings</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Title</string>
             </property>
             <property name="buddy">
              <cstring>subviewTitle</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="6">
            <widget class="QLineEdit" name="subviewTitle"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Left</string>
             </property>
             <property name="buddy">
              <cstring>subviewLeft</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="4">
            <widget class="QDoubleSpinBox" name="subviewLeft">
             <property name="suffix">
              <string>%</string>
             </property>
             <property name="decimals">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Top</string>
             </property>
             <property name="buddy">
              <cstring>subviewTop</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="6">
            <widget class="QDoubleSpinBox" name="subviewTop">
             <property name="suffix">
              <string>%</string>
             </property>
             <property name="decimals">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Right</string>
             </property>
             <property name="buddy">
              <cstring>subviewRight</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="4">
            <widget class="QDoubleSpinBox" name="subviewRight">
             <property name="suffix">
              <string>%</string>
             </property>
             <property name="decimals">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Bottom</string>
             </property>
             <property name="buddy">
              <cstring>subviewBottom</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="6">
            <widget class="QDoubleSpinBox" name="subviewBottom">
             <property name="suffix">
              <string>%</string>
             </property>
             <property name="decimals">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="3" column="4" colspan="2">
            <widget class="QLineEdit" name="subviewAlphaMask"/>
           </item>
           <item row="3" column="6">
            <widget class="QPushButton" name="subviewAlphaBrowseBtn">
             <property name="text">
              <string>Browse ...</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Brightness</string>
             </property>
             <property name="buddy">
              <cstring>subviewBrightnessSlider</cstring>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Contrast</string>
             </property>
             <property name="buddy">
              <cstring>subviewContrastSlider</cstring>
             </property>
            </widget>
           </item>
           <item row="5" column="4" colspan="2">
            <widget class="QSlider" name="subviewContrastSlider">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Hue</string>
             </property>
             <property name="buddy">
              <cstring>subviewHueSlider</cstring>
             </property>
            </widget>
           </item>
           <item row="6" column="4" colspan="2">
            <widget class="QSlider" name="subviewHueSlider">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Saturation</string>
             </property>
             <property name="buddy">
              <cstring>subviewSaturationSlider</cstring>
             </property>
            </widget>
           </item>
           <item row="7" column="4" colspan="2">
            <widget class="QSlider" name="subviewSaturationSlider">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="6">
            <widget class="QSpinBox" name="subviewBrightnessBox">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="4" column="4" colspan="2">
            <widget class="QSlider" name="subviewBrightnessSlider">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Alpha Mask</string>
             </property>
             <property name="buddy">
              <cstring>subviewAlphaMask</cstring>
             </property>
            </widget>
           </item>
           <item row="5" column="6">
            <widget class="QSpinBox" name="subviewContrastBox">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="6" column="6">
            <widget class="QSpinBox" name="subviewHueBox">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="7" column="6">
            <widget class="QSpinBox" name="subviewSaturationBox">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>subviewBrightnessSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>subviewBrightnessBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>797</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subviewBrightnessBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>subviewBrightnessSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>864</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>807</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subviewContrastSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>subviewContrastBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>748</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>878</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subviewContrastBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>subviewContrastSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>862</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>786</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subviewHueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>subviewHueBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>718</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>859</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subviewHueBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>subviewHueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>835</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subviewSaturationSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>subviewSaturationBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>772</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>853</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subviewSaturationBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>subviewSaturationSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>856</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>785</x>
     <y>562</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
